packages/kbn-i18n/src/core/formats.ts

```diff
@@ -74,68 +76,22 @@ export const formats: Formats = {
   },
   relative: {
     years: {
-      units: 'year',
+      style: 'long',
```

Comment:
FMI How do we know the relative time formats match the previous ones?

Reply:
The defaults are the same, you can also check by running Kibana and using the `FormattedRelativeTime` component